mappings/net/minecraft/server/filter/TextStream.mapping

```diff
@@ -14,7 +14,7 @@ CLASS net/minecraft/class_5513 net/minecraft/server/filter/TextStream
 			ARG 1 raw
 			ARG 2 filtered
 		METHOD method_33801 getRaw ()Ljava/lang/String;
-		METHOD method_33802 permitted (Ljava/lang/String;)Lnet/minecraft/class_5513$class_5837;
+		METHOD method_33802 unfiltered (Ljava/lang/String;)Lnet/minecraft/class_5513$class_5837;
```

Comment:
I recall this one is called if the api doesn't request any change after filtering as well

Reply:
This is the case. If the remote api permits a message it calls this factory method. A message is not "unfiltered" in that case
```suggestion
		METHOD method_33802 permitted (Ljava/lang/String;)Lnet/minecraft/class_5513$class_5837;
```